addons/udes_stock/models/stock_picking_type.py

```diff
@@ -1,8 +1,15 @@
-from odoo import models
+from odoo import models, fields
 
 
 class StockPickingType(models.Model):
     _inherit = "stock.picking.type"
 
+    u_auto_unlink_empty = fields.Boolean(
```

Comment:
I think we also need to add the new `u_push_on_drop` field here and also add it to the picking type form